source/stylesheets/refills/unstyled/_switch.scss

```diff
@@ -0,0 +1,50 @@
+.switch {  
+  $switch-width: 52px;
+  $switch-padding: 2px;
+  $switch-height: 32px;
+  $knob-size: $switch-height - ($switch-padding * 2);
+  $knob-width: $knob-size;
```

Comment:
Isn't there a risk in making the variables global? What if there's another `$switch-width` in the project? For some reason. The user might have created it somewhere else.